aws/modules_common.tf

```diff
@@ -29,10 +28,24 @@ module "codebuild_ci_cid-checker" {
   ]
 }
 
-module "codebuild_cd_cid-checker" {
-  count                    = local.is_mainnet_envs
+module "codebuild_cd_cid-checker_watcher_calibrationnet" {
+  count                    = local.is_dev_envs
+  source                   = "../modules/codebuild"
+  git_repository_name      = "cid-checker"
+  get_global_configuration = local.make_codebuild_global_configuration
+  privileged_mode          = true
+  is_build_concurrent      = false
+
+  depends_on = [
+    aws_secretsmanager_secret.github_cd_token_secret
+  ]
+}
+
+module "codebuild_cd_cid-checker_calibrationnet" {
+  count                    = local.is_dev_envs
   source                   = "../modules/codebuild"
   git_repository_name      = "cid-checker"
+  buildspec_logic          = file("${path.module}/templates/codebuild/deploy_cid_checker.yaml")
```

Comment:
I'm not sure we need this parameter here

Reply:
I have to use separate file because in other place you use the regular file for `watcher` component, so here I have to override it